Cargo.lock

```diff
@@ -2702,7 +2701,6 @@ dependencies = [
  "anyhow",
  "async-trait",
  "byte-unit",
- "chrono",
```

Comment:
Is there still a deps on chrono somewhere?
(I was expecting a larger diff in Cargo.lock)

Reply:
yes, on diesel. They use it for serialization and don't plan to replace it https://github.com/diesel-rs/diesel/issues/2928

Reply:
It seems optional though. Do we need that feature?